Automattic/wp-calypso
# Unpin dependencies

#### Changes proposed in this Pull Request

* Unpin some dependencies (similar to #42142, #42053, #41927, #41472, #41191)

#### Testing instructions

* Verify all tests pass
* Verify there are no new packages introduced: run `find . -type d -name node_modules | xargs rm -fr && yarn && NODE_OPTIONS=--max_old_space_size=8192 yarn run effective-module-tree -o list` in this branch and master (tested with commit fa6f0c10e6), save the output to a file and diff them.
